src/ROUTES.js

```diff
@@ -21,4 +21,10 @@ export default {
     DETAILS: 'details',
     DETAILS_WITH_LOGIN: 'details/:login',
     getDetailsRoute: login => `details/${login}`,
+    PARTICIPANTS: 'participants',
+    PARTICIPANTS_WITH_REPORT: 'participants/:reportID',
+    getParticipantsRoute: reportID => `participants/${reportID}`,
+    REPORT_ID: '/:reportID',
```

Comment:
This doesn't seem like any route we are using...

Reply:
Yeah the path prop didn't seem to help out there. Will remove this. Thanks for pointing out.